wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -60,53 +60,55 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
 - Triplets*
```

Comment:
This might be outside of your scope, but this footnote asterisk (and the one at line 77) has no footnote.

Reply:
Yeah, I'll need your help to add that footnote.

Reply:
Uh (this is out of my scope too)... reading some info from the old wiki, I think the footnote is supposed to say `*In musical terms, this is compound timing (rarely found in songs)`

Reply:
oh wait, it's says so two bullet points down...
I guess you could just remove the asterisks 🤷‍♂️

Reply:
Ah ok